--- pl-5.7.7/packages/clib/configure.in~	2009-02-26 15:19:36.000000000 +0100
+++ pl-5.7.7/packages/clib/configure.in	2009-03-10 18:20:37.000000000 +0100
@@ -120,8 +120,8 @@
 ( CC=$ACC; LD=$ALD; CPP=$ACPP; CFLAGS=$CFLAGS; LDFLAGS=$LDFLAGS;
   export CC LD CFLAGS LDFLAGS
 
-  (cd maildrop/rfc822 && $SHELL configure)
-  (cd maildrop/rfc2045 && $SHELL configure)
+  (cd maildrop/rfc822 && $SHELL ./configure)
+  (cd maildrop/rfc2045 && $SHELL ./configure)
 )
 AC_MSG_RESULT("Done")
 
