--- pl-3.4.0/src/Makefile.in~	Wed Jul 26 23:14:39 2000
+++ pl-3.4.0/src/Makefile.in	Fri Oct 20 17:16:09 2000
@@ -48,7 +48,7 @@
 lmansec=3
 mandir=$(man_prefix)/man$(mansec)
 lmandir=$(man_prefix)/man$(lmansec)
-plmandir=$(PLBASE)/man
+plmandir=$(man_prefix)/man1
 
 CC=@CC@
 LIBS=@LIBS@
@@ -238,34 +238,34 @@
 dv-install:	install-arch install-libs install-bins
 
 install-mt:	
-		$(INSTALL_PROGRAM) $(PL) $(PLBASE)/bin/$(ARCH)/pl-mt
-		$(REL_LN) $(PLBASE)/bin/$(ARCH)/pl-mt $(bindir)
+		$(INSTALL_PROGRAM) $(PL) $(DESTDIR)$(PLBASE)/bin/$(ARCH)/pl-mt
+		$(REL_LN) $(PLBASE)/bin/$(ARCH)/pl-mt $(DESTDIR)$(bindir)
 
 install-bins:	install-arch
-		-if [ ! -d $(bindir)  ]; then $(MKDIR) $(bindir); fi
+		-if [ ! -d $(DESTDIR)$(bindir)  ]; then $(MKDIR) $(DESTDIR)$(bindir); fi
 		for f in $(PL) plld plrc; do \
-		    $(REL_LN) $(PLBASE)/bin/$(ARCH)/$$f $(bindir); \
+		    $(REL_LN) $(PLBASE)/bin/$(ARCH)/$$f $(DESTDIR)$(bindir); \
 		done
 
 install-vbins:	install-arch
-		-if [ ! -d $(bindir)  ]; then $(MKDIR) $(bindir); fi
-		$(REL_LN) $(PLBASE)/bin/$(ARCH)/$(PL) $(bindir)/$(PLV)
+		-if [ ! -d $(DESTDIR)$(bindir)  ]; then $(MKDIR) $(DESTDIR)$(bindir); fi
+		$(REL_LN) $(PLBASE)/bin/$(ARCH)/$(PL) $(DESTDIR)$(bindir)/$(PLV)
 		for f in plld plrc; do \
-		    $(REL_LN) $(PLBASE)/bin/$(ARCH)/$$f $(bindir); \
+		    $(REL_LN) $(PLBASE)/bin/$(ARCH)/$$f $(DESTDIR)$(bindir); \
 		done
 
 install-arch:	idirs iruntime
 		for f in $(PL) plld rc/plrc; do \
-		    $(INSTALL_PROGRAM) $$f $(PLBASE)/bin/$(ARCH); \
+		    $(INSTALL_PROGRAM) $$f $(DESTDIR)$(PLBASE)/bin/$(ARCH); \
 		done
 
 install-libs:	idirs iboot ilib iruntime iinclude iman
-		$(INSTALL_DATA) $(STARTUPPATH) $(PLBASE)/$(BOOTFILE)
-		$(INSTALL_DATA) swipl $(PLBASE)
-		$(INSTALL_DATA) swiplbin $(PLBASE)/bin/swipl
+		$(INSTALL_DATA) $(STARTUPPATH) $(DESTDIR)$(PLBASE)/$(BOOTFILE)
+		$(INSTALL_DATA) swipl $(DESTDIR)$(PLBASE)
+		$(INSTALL_DATA) swiplbin $(DESTDIR)$(PLBASE)/bin/swipl
 		# $(INSTALL_DATA) README.bin $(PLBASE)
 		# $(INSTALL_DATA) ../COPYING $(PLBASE)
-		cd $(PLBASE)/library; \
+		cd $(DESTDIR)$(PLBASE)/library; \
 		   ../bin/$(ARCH)/$(PL) \
 			-f none \
 			-g make_library_index\('.'\) \
@@ -275,58 +275,58 @@
 		include runtime runtime/$(ARCH)
 
 iruntime:
-		if [ ! -d $(PLBASE)/runtime/$(ARCH) ]; then \
-			$(MKDIR) $(PLBASE)/runtime/$(ARCH); \
+		if [ ! -d $(DESTDIR)$(PLBASE)/runtime/$(ARCH) ]; then \
+			$(MKDIR) $(DESTDIR)$(PLBASE)/runtime/$(ARCH); \
 		fi
-		if [ -f $(RUNTIME) ]; then \
-			$(INSTALL_DATA) $(RUNTIME) $(PLBASE)/runtime/$(ARCH); \
+		if [ -f $(DESTDIR)$(RUNTIME) ]; then \
+			$(INSTALL_DATA) $(RUNTIME) $(DESTDIR)$(PLBASE)/runtime/$(ARCH); \
 		fi
 
 idirs:
-		-if [ ! -d $(prefix)/lib  ];    then $(MKDIR) $(prefix)/lib;     fi
-		-if [ ! -d $(PLBASE)  ];    then $(MKDIR) $(PLBASE); 	fi
+		-if [ ! -d $(DESTDIR)$(prefix)/lib  ];    then $(MKDIR) $(DESTDIR)$(prefix)/lib;     fi
+		-if [ ! -d $(DESTDIR)$(PLBASE)  ];    then $(MKDIR) $(DESTDIR)$(PLBASE); 	fi
 		for d in $(installdirs); do \
-			if [ ! -d $(PLBASE)/$$d ]; then \
-				$(MKDIR) $(PLBASE)/$$d; \
+			if [ ! -d $(DESTDIR)$(PLBASE)/$$d ]; then \
+				$(MKDIR) $(DESTDIR)$(PLBASE)/$$d; \
 			fi; \
 		done
 
 
 iboot:		
 		cd ../boot; \
-		for f in *.pl; do $(INSTALL_DATA) $$f $(PLBASE)/boot; done
+		for f in *.pl; do $(INSTALL_DATA) $$f $(DESTDIR)$(PLBASE)/boot; done
 ilib:		
 		cd ../library; \
 		for f in $(PLLIBS); do \
-			$(INSTALL_DATA) $$f $(PLBASE)/library; \
+			$(INSTALL_DATA) $$f $(DESTDIR)$(PLBASE)/library; \
 		done
 
 iinclude:       
 		cd ../include; \
 		for f in SWI-Prolog.h SWI-Exports SWI-Stream.h stub.c; do \
 		    if [ -f $$f ]; then \
-			$(INSTALL_DATA) $$f $(PLBASE)/include; \
+			$(INSTALL_DATA) $$f $(DESTDIR)$(PLBASE)/include; \
 		    fi; \
 		done
-		if [ -d $(prefix)/include -a -w $(prefix)/include ]; then \
-		    $(INSTALL_DATA) ../include/SWI-Prolog.h $(prefix)/include; \
+		if [ -d $(DESTDIR)$(prefix)/include -a -w $(DESTDIR)$(prefix)/include ]; then \
+		    $(INSTALL_DATA) ../include/SWI-Prolog.h $(DESTDIR)$(prefix)/include; \
 		fi
 
 iman:		@INSTALLRLMAN@
-		$(INSTALL_DATA) pl.1 $(plmandir)/$(PL).$(mansec)
-		$(INSTALL_DATA) $(srcdir)/plrc.1 $(plmandir)/plrc.$(mansec)
-		$(INSTALL_DATA) $(srcdir)/plld.1 $(plmandir)/plld.$(mansec)
-		$(INSTALL_DATA) $(srcdir)/readline.3 $(plmandir)/readline.$(lmansec)
-		-if [ ! -d $(man_prefix) ]; then $(MKDIR) $(man_prefix); fi
-		-if [ ! -d $(mandir)  ];    then $(MKDIR) $(mandir);     fi
-		-$(INSTALL_DATA) $(plmandir)/$(PL).$(mansec) $(mandir)
-		-$(INSTALL_DATA) $(plmandir)/plrc.$(mansec) $(mandir)
-		-$(INSTALL_DATA) $(plmandir)/plld.$(mansec) $(mandir)
+		$(INSTALL_DATA) pl.1 $(DESTDIR)$(plmandir)/$(PL).$(mansec)
+		$(INSTALL_DATA) $(srcdir)/plrc.1 $(DESTDIR)$(plmandir)/plrc.$(mansec)
+		$(INSTALL_DATA) $(srcdir)/plld.1 $(DESTDIR)$(plmandir)/plld.$(mansec)
+		$(INSTALL_DATA) $(srcdir)/readline.3 $(DESTDIR)$(plmandir)/readline.$(lmansec)
+		-if [ ! -d $(man_prefix) ]; then $(MKDIR) $(DESTDIR)$(man_prefix); fi
+		-if [ ! -d $(mandir)  ];    then $(MKDIR) $(DESTDIR)$(mandir);     fi
+		-$(INSTALL_DATA) $(plmandir)/$(PL).$(mansec) $(DESTDIR)$(mandir)
+		-$(INSTALL_DATA) $(plmandir)/plrc.$(mansec) $(DESTDIR)$(mandir)
+		-$(INSTALL_DATA) $(plmandir)/plld.$(mansec) $(DESTDIR)$(mandir)
 
 install_rlman:
-		-if [ ! -d $(lmandir) ]; then $(MKDIR) $(lmandir); fi
-		-if [ ! -r $(lmandir)/readline.$(lmansec) ]; then \
-		  $(INSTALL_DATA) $(srcdir)/readline.3 $(lmandir)/readline.$(lmansec); \
+		-if [ ! -d $(DESTDIR)$(lmandir) ]; then $(MKDIR) $(DESTDIR)$(lmandir); fi
+		-if [ ! -r $(DESTDIR)$(lmandir)/readline.$(lmansec) ]; then \
+		  $(INSTALL_DATA) $(srcdir)/readline.3 $(DESTDIR)$(lmandir)/readline.$(lmansec); \
 		fi
 
 #
@@ -342,17 +342,17 @@
 		@echo "Creating runtime environment in $(RTHOME)"
 		@echo "****************"
 		@for d in $(RTDIRS); do \
-		    if [ ! -d $$d ]; then \
-			echo $(MKDIR) $$d; \
-			$(MKDIR) $$d; \
+		    if [ ! -d $(DESTDIR)$$d ]; then \
+			echo $(MKDIR) $(DESTDIR)$$d; \
+			$(MKDIR) $(DESTDIR)$$d; \
 		    fi; \
 		done
-		$(INSTALL_PROGRAM) $(PL) $(RTHOME)/bin/$(ARCH)/pl
-		$(INSTALL_DATA) $(srcdir)/pl.1 $(RTHOME)/man/pl.1
-		$(INSTALL_DATA) swipl $(RTHOME)/swipl
-		$(INSTALL_DATA) swiplbin $(RTHOME)/bin/swipl
-		$(INSTALL_DATA) $(srcdir)/README.RT $(RTHOME)
-#		-cd $(bindir); rm -f plrt; ln -s $(RTHOME)/bin/$(ARCH)/pl plrt
+		$(INSTALL_PROGRAM) $(PL) $(DESTDIR)$(RTHOME)/bin/$(ARCH)/pl
+		$(INSTALL_DATA) $(srcdir)/pl.1 $(DESTDIR)$(RTHOME)/man/pl.1
+		$(INSTALL_DATA) swipl $(DESTDIR)$(RTHOME)/swipl
+		$(INSTALL_DATA) swiplbin $(DESTDIR)$(RTHOME)/bin/swipl
+		$(INSTALL_DATA) $(srcdir)/README.RT $(DESTDIR)$(RTHOME)
+#		-cd $(DESTDIR)$(bindir); rm -f plrt; ln -s $(RTHOME)/bin/$(ARCH)/pl plrt
 
 #
 # Cleanup
