--- pl-6.0.2/packages/xpce/src/Makefile.in.orig	2012-02-09 16:01:49.000000000 +0100
+++ pl-6.0.2/packages/xpce/src/Makefile.in	2012-03-17 09:46:12.973414463 +0100
@@ -708,7 +708,7 @@
 		    fi; \
 		    echo done; \
 		done
-		$(PL) -C "$(DESTDIR)$(DVHOME)" -f none -t 'halt(1)' \
+		./xpce.sh -C "$(DESTDIR)$(DVHOME)" -f none -t 'halt(1)' \
 			-g "make_library_index('prolog/lib'),halt"
 		$(INSTALL_DATA) ../prolog/lib/Overview $(DESTDIR)$(DVHOME)/prolog/lib
 		$(INSTALL_DATA) ../prolog/lib/trace/pltracer.hlp $(DESTDIR)$(DVHOME)/prolog/lib/trace
